src/components/HypothesisTesting/PerformTest.js

```diff
@@ -21,23 +22,60 @@ export default function PerformTest({ distType, shape, tails, mue0 }) {
     setPopArr(dataFromDistribution(shape, 2000, { mean: 69, low: 59, hi: 79 }))
   }, [shape]);
 
+/*
+  const Pop2Mean = Math.floor(Math.random() * (66 - 61 + 1)) + 61;
```

Comment:
You should be able to do `random(min, max)`, which will return a random number between `min` and `max`.

Reply:
Also, I would rename this variable to use camelCase, which matches the naming standard throughout the project.